_data/taps/versions/bing-ads.yml

```diff
@@ -0,0 +1,161 @@
+# -------------------------- #
+#      BING ADS VERSIONS     #
+# -------------------------- #
+
+latest-version: "2.0"
+
+released-versions:
+  - number: "2.0"
+    date-released: "October 4, 2018"
+    # date-last-connection:
+    deprecated: false
+    deprecation-date: "n/a"
+
+  - number: "1.0"
+    date-released: ""
+    date-last-connection: "October 3, 2018"
+    deprecated: true
+    deprecation-date: ""
```

Comment:
I guess this would be today? (Oct 4) Unfortunately that doesn't give us our nice window, but I guess it's technically true...

Reply:
Sigh, corrected :|